codex-ui/src/vue/card/Card.vue

```diff
@@ -9,22 +9,29 @@
       >
       <div :class="[$style.updatedAt, 'text-ui-subtle']">{{ updatedAt }}</div>
     </div>
+    <Button size="medium">{{ buttonText }}</Button>
   </div>
 </template>
 
 <script setup lang="ts">
 import { defineProps } from 'vue';
 import Heading from '../heading/Heading.vue';
+import Button from '../button/Button.vue';
+import type { CardOrientation } from './Card.types';
 
 /**
  * NoteListItem props, receive Note
  *
  * @param {string} title - Note title
  * @param {string} updatedAt - Last updated date of note
+ * @param {string} buttonText - Button text
+ * @param orientation - Orientation of card
  */
 defineProps<{
   title: string;
   updatedAt: string;
+  buttonText: string;
```

Comment:
button should be optional. And how the click listeners should work? Maybe use `slot` instead of button? So user can pass `Button` or `RouterLink` or something else from outside